src/gui/widgets/template_list_widget.cpp

```diff
@@ -1064,6 +1060,27 @@ void TemplateListWidget::importClicked()
 	}
 }
 
+void TemplateListWidget::releaseGeorefClicked()
+{
+	auto temp = getCurrentTemplate();
+	if (!temp || !temp->isTemplateGeoreferenced())
+		return;
+	
+	temp->releaseGeoreference();
```

Comment:
The remaining code might be better replaced by calling `updateButtons()`. And with that change, it could be clearer to invert the operation of th preceding `if` guard.